samples/cellular/nrf_provisioning/sample.yaml

```
@@ -4,5 +4,5 @@ tests:
   sample.cellular.nrf_provisioning:
     build_only: true
     integration_platforms:
-      - nrf9160dk_nrf9160_ns
-    platform_allow: nrf9160dk_nrf9160_ns
+      - nrf9161dk_nrf9161_ns
+    platform_allow: nrf9161dk_nrf9161_ns
```

Shouldn't this sample support both nRF9160DK and nRF9161DK?
```suggestion
    platform_allow: nrf9160dk_nrf9160_ns nrf9161dk_nrf9161_ns
```
nRF9160 could also be added to integration platforms to build both boards in CI.

---

In my understanding device provisioning is not supported on 9160dk. @plskeggs should the sample support both boards?